tests/testthat/test-undesirable_function_linter.R

```diff
@@ -7,7 +7,7 @@ test_that("linter returns correct linting", {
 
   expect_lint("x <- options()", NULL, linter)
   expect_lint("cat(\"Try to return\")", NULL, linter)
-  expect_lint("lapply(x, log10)", list(message=msgL, line_number=1L, column_number=11L), linter)
+  #expect_lint("lapply(x, log10)", list(message=msgL, line_number=1L, column_number=11L), linter)
```

Comment:
I think it is worth this tradeoff, i.e. to let go of this use case and gain the ability to use the undesirable function names in other contexts, i.e. as a object name.

Reply:
i would add an option to the undesirable_function_linter signature instead (maybe strict=TRUE by default?) which toggles whether to include SYMBOLs

Reply:
I agree having an option to toggle the existing behavior seems worthwhile.

Reply:
Alright, I'll split the GHA logging stuff off into a separate PR and implement this as a switch.

I am using this linter with usethis now, to enforce "fs everywhere!", so I'd love to get to the point where I can use dev lintr, instead of a branch on my fork.